pkg/http/headers.go

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) 2015-2021 MinIO, Inc.
```

Comment:
```suggestion
// Copyright (c) 2022 MinIO, Inc.
```

and so many more hahaha